packages/headless/src/controllers/core/search-box/headless-core-search-box-options.ts

```diff
@@ -45,11 +45,10 @@ export interface SearchBoxOptions {
 
 type DefaultSearchBoxOptions = Pick<
   SearchBoxOptions,
-  'enableQuerySyntax' | 'numberOfSuggestions' | 'clearFilters'
+  'numberOfSuggestions' | 'clearFilters'
 >;
 
 export const defaultSearchBoxOptions: Required<DefaultSearchBoxOptions> = {
-  enableQuerySyntax: false,
```

Comment:
One thing that's important to keep is that query syntax should be disabled by default for 99% of use cases.

We need to make it so that implementers have to go "out of their way" to have a search box where query syntax is enabled. We don't want to end up with implementers having to always specify that they want query syntax disabled.

Leaving the property `undefined` when it's sent to the API (when the option is not set explicitly on the controller) means that query syntax will be enabled at the API level. We do not want that.

I think it would be better to add an option on atomic search box.

As for why we want query syntax disabled in the vast majority of case, it's because query syntax is "all or nothing". And when you submit some special characters, the query syntax will kick in at the API and index level, and it's easy to get 4XX errors due to invalid query syntax.

For most of our search interface that will be used by "normal non Coveo knowledgeable" end user, it's easy to see that someone could add `[` or `(`, `@` etc. characters in the search box without knowing that they are query syntax sensitive character, and they would get back a cryptic error with no way of knowing how to fix it.

Said differently, query syntax in the search box is a power user feature.